tests/tests_certificate2.yml

```diff
@@ -0,0 +1,62 @@
+---
```

Comment:
If we drop the old approach, you can update the existing tests_certificate.yml, and avoid the "huh, what's 2" duplication.

Reply:
It'd be nice to keep the original tests_certificate.yml, IMHO. I can rename it to, like, tests_certificate_internal.yml or something similar. (maybe, I can rename the original tests_certificate.yml to tests_certificate_external.yml, as well. :)

Reply:
Yes, with a renaming it's clearer what they do.